packages/flutter/lib/src/material/ink_splash.dart

```diff
@@ -75,6 +75,50 @@ class _InkSplashFactory extends InteractiveInkFeatureFactory {
   }
 }
 
+abstract class InkSplashRippleMixin {
+  // This class is intended to be used as a mixin, and should not be
+  // extended directly.
+  // Reduces duplicate code between InkSplash and InkRipple 
```

Comment:
Line 81 seems to fail the analyze test "ink_splash.dart:81: trailing U+0020 space character": https://cirrus-ci.com/task/5116804708171776?command=main#L8